src/test/java/uk/co/ramp/covid/simulation/ModelTest.java

```diff
@@ -21,43 +21,43 @@
 
 public class ModelTest extends SimulationTest {
 
-    int population;
-    int nInfections;
-    int nIter;
-    int nDays;
-    int RNGSeed;
-
-    @Before
-    public void setupParams() {
-        population = 10000;
-        nInfections = 10;
-        nIter = 1;
-        nDays = 60;
-        RNGSeed = 402;
-    }
+    private static int population = 10000;
+    private static int nInfections = 150;
+    private static int nIter = 1;
+    private static int nDays = 60;
+    private static int RNG = 402;
 
-    @Test
-    public void testBaseLine() {
-        final int inf = nInfections * 5;
+    // To avoid recomputes
+    private static List<List<DailyStats>> cachedStats = null;
+
+    @BeforeClass
+    public static void singleRun() throws IOException {
+        ParameterIO.readParametersFromFile("parameters/example_population_params.json");

```

Comment:
If you're doing this here, and reseeding the RNG, then maybe should no longer inherit from SimulationTest ?